roles/ceph-rgw-loadbalancer/tasks/pre_requisite.yml

```diff
@@ -41,12 +41,12 @@
   block:
     - name: set_fact rgw_ports
       set_fact:
-        rgw_ports: "{{ rgw_ports | default([]) + [hostvars[item]['rgw_instances']['radosgw_frontend_port']] | unique }}"
+        rgw_ports: "{{ rgw_ports | default([]) + hostvars[item].rgw_instances|map(attribute='radosgw_frontend_port')|list }}"
```

Comment:
```suggestion
        rgw_ports: "{{ rgw_ports | default([]) | union(hostvars[item]['rgw_instances'] | map(attribute='radosgw_frontend_port') | map('string') | list) }}"
```

Reply:
Thanks for the suggestion, I've amended the commit accordingly!